src/plugins/vis_type_vega/public/expressions/helpers.test.js

```diff
@@ -21,10 +21,25 @@ import {
   simpleDimensions,
   noXAxisDimensions,
 } from './__mocks__';
+import {
+  TEST_DATATABLE_NO_VIS_LAYERS,
+  TEST_DATATABLE_NO_VIS_LAYERS_DIRTY,
+} from '../../../vis_augmenter/public/test_constants';
 
 describe('helpers', function () {
+  describe('formatDatatable()', function () {
+    it('formatSimpleDatatable', function () {
+      expect(formatDatatable(TEST_DATATABLE_NO_VIS_LAYERS)).toBe(TEST_DATATABLE_NO_VIS_LAYERS);
+    });
+    it('formatDirtyDatatable', function () {
+      expect(formatDatatable(TEST_DATATABLE_NO_VIS_LAYERS_DIRTY)).toStrictEqual(
+        TEST_DATATABLE_NO_VIS_LAYERS
+      );
+    });
+  });
+
   describe('cleanString()', function () {
-    it('string should not contain "', function () {
+    it('string should not contain', function () {
       const dirtyString = '"someString"';
       expect(cleanString(dirtyString)).toBe('someString');
     });
```

Comment:
nit: should the test name still say `'string should not contain "'`?

Reply:
or `string should not contain quotation marks`?

Reply:
Good catch, I am changing it to quotation marks to ensure another dev doesn't accidentally remove `"` thinking it was a mistake.